### ISSUE TITLE
Deploy python scripting framework

### ISSUE THREAD

**Issue by sadmansk** [uwnrg/minotaur-cpp#14]
~~*More details to come*~~

There are two parts to this, the first is the logic part, which would contain two interface files, one C++ (which exposes the C++ functions to Python) and one Python (which provides helper functions to the scripts). The second part would be the UI, which could be worked on separately from the first one. The goal for the UI is to have the ability to open a new window where you can open `.py` files, or write `.py` files and the ability to run them. So this is the list of tasks that need to be done:

### GUI
- [x] Openning and saving files
- [X] Text input
- [x] Send the text body from the text field to the script engine

### Engine
- [x] Interprete string as a python program
- [x] Move controller based on the interpretation

**Comment by Mogball** [comment]
We can check off 'Send the text body..' and both Engine features after #37 is merged. I'll handle opening and saving files in a separate PR

**Comment by sadmansk** [comment]
Closed by #51
